components/dashboard/src/teams/Members.tsx

```diff
@@ -206,7 +206,7 @@ export default function() {
             <div className="border-t border-b border-gray-200 dark:border-gray-800 -mx-6 px-6 py-4 flex flex-col">
                 <label htmlFor="inviteUrl" className="font-medium">Invite URL</label>
                 <div className="w-full relative">
-                    <input name="inviteUrl" disabled={true} readOnly={true} type="text" value={getInviteURL(genericInvite.id)} className="rounded-md w-full truncate pr-8" />
+                    <input name="inviteUrl" disabled={true} readOnly={true} type="text" value={getInviteURL(genericInvite.id)} className="rounded-md overflow-x-scroll w-full truncate pr-8" />
```

Comment:
Thanks for the contribution! I think you'll need to remove the `truncate` here?

Reply:
> I think you'll need to remove the `truncate` here?

Oh Yes. We have to. Missed out on it. Let me change this.